### ISSUE TITLE
failed to build for freebsd_amd64_v1

### ISSUE THREAD

**Issue by stv0g** [pion/ice#482]
```
Error: mod/github.com/pion/ice/v2@v2.2.10/udpmsghelper.go:18:73: undefined: syscall.IPV6_2292PKTINFO
Error: mod/github.com/pion/ice/v2@v2.2.10/udpmsghelper.go:21:71: undefined: syscall.IP_PKTINFO
Error: mod/github.com/pion/ice/v2@v2.2.10/udpmsghelper.go:34:63: undefined: syscall.IP_PKTINFO
Error: mod/github.com/pion/ice/v2@v2.2.10/udpmsghelper.go:35:26: undefined: syscall.Inet4Pktinfo
Error: mod/github.com/pion/ice/v2@v2.2.10/udpmsghelper.go:40:65: undefined: syscall.IPV6_2292PKTINFO
```

**Comment by Sean-Der** [comment]
@boks1971 @cnderrauber 

We probably need to not support multi IP interfaces for UDPMux. I don’t know how to effectively test all the platforms on this one.

**Comment by cnderrauber** [comment]
@Sean-Der @stv0g low-level API is not reliable and have compatible problems on some platforms, will draft a new implement to remove those low-level API dependency.

**Comment by stv0g** [comment]
I submitted a PR which fixes this.
The main issue is that the build tags have been set to broad.
The OOB PKTINFO socket options are now only used for Linux and other platforms will be ignored.
Support for other platforms should be added later one by one

**Comment by stv0g** [comment]
This will be fixed by merging #483
